addons/binding/org.openhab.binding.onebusaway/src/main/java/org/openhab/binding/onebusaway/OneBusAwayBindingConstants.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
Luckily it was only my binding that needed updating.  It was non-obvious that that command had to be ran from the top level though.